design/use_cases/outdoor_navigation_use_case.md

```diff
@@ -0,0 +1,14 @@
+# Outdoor Navigation
+As a Robot user I want my robot to autonomously navigate to a given location on a given outdoor map, such as a college campus or street, so that it can do something useful at that location
```

Comment:
In these use cases, we keep referring to navigating to a specific location. Should we include navigating to that position with a specific orientation? It's not really an issue for a turtlebot, but for a car-like robot it matters.

Reply:
I think it's a good idea to generalize, where appropriate (as in this case).

Reply:
@crdelsey - I think it will be necessary to specify a location and pose (orientation), but also agree with @mjeronimo that we keep it general here if possible